CHANGELOG.md

```diff
@@ -32,6 +56,107 @@ Now we'll checking for links in Pending state, and if they haven't hit a configu
 
 The new config property is `pendingLinkTimeoutSeconds` in the controller config file under `network`, and defaults to 10 seconds.
 
+## Enhanced CLI Login Functionality
+### Server Trust
+#### Untrusted Servers
+If you don't provide a certificates file when logging in, the server's well known certificates will now be pulled from the server and you will be prompted if you want to use them. 
+If certs for the host have previously been retrieved they will be used. Certs stored locally will be checked against the certs on the server when logging in. 
+If a difference is found, the user will be notified and asked if they want to update the local certificate cache.
+
+If you provide certificates during login, the server's certificates will not be checked or downloaded. Locally cached certificates for that host will not be used.
+
+#### Trusted Servers
+If working with a server which is using certs that your OS already recognizes, nothing will change. No cert needs to be provided and the server's well known certs will not be downloaded.
+
+### Identities
+The Ziti CLI now suports multiple identities. An identity can be specified using `--cli-identity` or `-i`. 
+
+Example commands:
+```
+$ ziti edge login -i dev localhost:1280
+Enter username: admin
+Enter password: 
+Token: 76ff81b4-b528-4e2c-ad73-dcb0a39b6489
+Saving identity 'dev' to ~/.config/ziti/ziti-cli.json
```

Comment:
should we decide on 'a standard' to follow? `~/.config/ziti` or `~/.ziti'`?

Reply:
It uses `.config/ziti` on linux, since that's linux standard, and `.ziti` elsewhere. It can be overridden with ZITI_HOME.